apps/src/blockly/customBlocks/cdoBlockly/spritelabBlocks.js

```diff
@@ -98,4 +99,184 @@ export const blocks = {
         break;
     }
   },
+
+  installBehaviorBlocks(behaviorEditor) {
+    const generator = Blockly.getGenerator();
```

Comment:
Can we extract this logic out into another file? It's a bunch of code and this file will be more readable if it doesn't contains super-specific logic.

Reply:
Which logic do you think it makes the most sense to move? The code for `installBehaviorBlocks`?

Reply:
yeah we could just pull all the code out (and pass in a generator?) or pull out functions/objects that get assigned to various fields on `generator`. Basically I want to avoid making this another mega-file if we can avoid it, but I don't have a super strong opinion on the specific refactoring here.

Reply:
Assuming you're okay with tracking this as a follow-up, I added it here: https://codedotorg.atlassian.net/browse/CT-13